camelot/data/types/__init__.py

```diff
@@ -98,3 +100,246 @@ def get_by_id(self, type_id):
     sensitivity_level_type(5, 'sensitive_health',    'Data field containing sensitive health information'),
 )
 
+# Zip/Postal code types with regular expression specifications as required for use with BECRIS.
+class zip_code_type(collections.namedtuple('national_identifier_type', ('code', 'regex', 'repl', 'example'))):
+
+    @property
+    def id(self):
+        return self.code
+
+    @property
+    def name(self):
+        return self.code
+
+    @property
+    def prefixes(self):
+        prefixes = []
+        if self.code is not None:
+            prefixes.append(self.code)
+        if self.regex is not None:
+            m = re.match('\(\?\:([^\)]+)\)', self.regex)
+            if m is not None:
+                prefixes.extend(m.group(1).split('|'))
+        return set(prefixes)
+
+    @property
+    def compact_repl(self):
```

Comment:
worden die compact en format repl ergens gebruikt ?

Reply:
Voorlopig niet, dat is een artefact dat is overgenomen van de national identifier types. Maar kan mss op termijn nuttig zijn op documenten.

Reply:
ok, vreesde dat ik ergens iets ah missen was :)